lib/analytics.js

```diff
@@ -114,13 +126,18 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     if (!Integration) delete settings[name];
   }, settings);
 
-  // add integrations
-  each(function(opts, name) {
-    var Integration = self.Integrations[name];
-    var integration = new Integration(clone(opts));
-    self.log('initialize %o - %o', name, opts);
-    self.add(integration);
-  }, settings);
+  if (doNotTrack) {
+    // Clear manually added integrations
+    this._integrations = {};
```

Comment:
@Rowno @f2prateek @tsholmes now im confused. Is this value meant to stop us from collecting data even for ourselves? 

Like prateek said somewhere else,

> Yep. I think the key to document for downstream integrations here is that this field is meant for "analytics" and not really about deciding whether to accept/reject this event.

What analytics? You wouldn't receive any data if this doNotTrack value was set to true. So it might lead to confusion where people might be like well im missing data in my warehouse because data where doNotTrack was enabled would not even be collected